kibble/render/language_renderer.go

```diff
@@ -0,0 +1,72 @@
+//    Copyright 2018 SHIFT72
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package render
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"kibble/models"
+)
+
+// Setup and Create language files based on API or local language files.
+func WriteLanguageFiles(site *models.Site, sourcePath string) error {
+	if !site.Toggles["translations_api"] {
+		return nil
```

Comment:
```suggestion
		// Do nothing and use the language files provided by the repo, e.g. en_AU.all.json
		return nil
```

Reply:
I removed this translation toggle check, and placed the function call inside check block in render.go